Userland/Libraries/LibJS/Print.cpp

```diff
@@ -165,6 +165,7 @@ ErrorOr<void> print_array(JS::PrintContext& print_context, JS::Array const& arra
 {
     TRY(js_out(print_context, "["));
     bool first = true;
+    size_t cnt = 0;
```

Comment:
We like to use descriptive variable names, something like `printed_count` would be better. (See [CodingStyle.md](https://github.com/LadybirdBrowser/ladybird/blob/master/Documentation/CodingStyle.md))